install/operator/vendor/modules.txt

```diff
@@ -360,9 +360,9 @@ k8s.io/apimachinery/pkg/runtime/serializer
 k8s.io/apimachinery/pkg/runtime/serializer/json
 k8s.io/apimachinery/pkg/runtime/serializer/versioning
 k8s.io/apimachinery/pkg/util/json
-k8s.io/apimachinery/pkg/api/meta
 k8s.io/apimachinery/pkg/labels
 k8s.io/apimachinery/pkg/api/resource
+k8s.io/apimachinery/pkg/api/meta
```

Comment:
No idea where the changes to this file came from

Reply:
I don't think so, I haven't opened that file and I edited the others in vim, so no automatic reformating. Is it possible the go compiler reformatted it? I'll revert it back anyway, just wondering